### PR TITLE
add cloud doc url and periods

### PR DESCRIPTION
**Pull request by Icemap**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [ ] master (the latest development version)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by qiancai** [comment]
@Icemap Please update the content based on the development branch. Thanks.

**Comment by Icemap** [comment]
Got it.

File: develop/bookshop-schema-design.md [hunk 1]
```diff
@@ -91,6 +91,8 @@ tiup demo bookshop prepare --users=200000 --books=500000 --authors=100000 --rati
 
    > table(s) [`bookshop`.`authors`, `bookshop`.`book_authors`, `bookshop`.`books`, `bookshop`.`orders`, `bookshop`.`ratings`, `bookshop`.`users`] are not empty.
 
+7. 你可以通过 [TiDB Cloud 文档](https://docs.pingcap.com/tidbcloud/tidb-cloud-quickstart) 获取更多有关 TiDB Cloud 的信息。
```
Comment (Oreoxmt) [inline develop/bookshop-schema-design.md:94]:
```suggestion
7. 你可以通过 [TiDB Cloud 文档](https://docs.pingcap.com/tidbcloud) 获取更多有关 TiDB Cloud 的信息。
```

File: develop/build-cluster-in-cloud.md [hunk 2]
```diff
@@ -6,39 +6,39 @@ title: 使用 TiDB Cloud(DevTier) 构建 TiDB 集群
 
 # 使用 TiDB Cloud(DevTier) 构建 TiDB 集群
 
-本页将指导您以最快的方式开始使用 TiDB 。您将使用 TiDB Cloud ，创建并启动一个免费的 TiDB 集群，使用 TiDB SQL 客户端，插入数据。随后将从示例程序读取出数据。
+本页将指导您以最快的方式开始使用 TiDB 。您将使用 [TiDB Cloud](https://docs.pingcap.com/tidbcloud/tidb-cloud-quickstart) ，创建并启动一个免费的 TiDB 集群，使用 TiDB SQL 客户端，插入数据。随后将从示例程序读取出数据。
```
Comment (Oreoxmt) [inline develop/build-cluster-in-cloud.md:9]:
```suggestion
本页将指导您以最快的方式开始使用 TiDB 。您将使用 [TiDB Cloud](https://en.pingcap.com/tidb-cloud) ，创建并启动一个免费的 TiDB 集群，使用 TiDB SQL 客户端，插入数据。随后将从示例程序读取出数据。
```

File: develop/bookshop-schema-design.md [hunk 3]
```diff
@@ -91,6 +91,8 @@ tiup demo bookshop prepare --users=200000 --books=500000 --authors=100000 --rati
 
    > table(s) [`bookshop`.`authors`, `bookshop`.`book_authors`, `bookshop`.`books`, `bookshop`.`orders`, `bookshop`.`ratings`, `bookshop`.`users`] are not empty.
 
+7. 你可以通过 [TiDB Cloud 文档](https://docs.pingcap.com/tidbcloud) 获取更多有关 TiDB Cloud 的信息。
```
Comment (Oreoxmt) [inline develop/bookshop-schema-design.md:94]:
```suggestion
7. 你可以通过 [TiDB Cloud 文档](https://docs.pingcap.com/tidbcloud)获取更多有关 TiDB Cloud 的信息。
```

File: develop/build-cluster-in-cloud.md [hunk 4]
```diff
@@ -6,39 +6,39 @@ title: 使用 TiDB Cloud(DevTier) 构建 TiDB 集群
 
 # 使用 TiDB Cloud(DevTier) 构建 TiDB 集群
 
-本页将指导您以最快的方式开始使用 TiDB 。您将使用 TiDB Cloud ，创建并启动一个免费的 TiDB 集群，使用 TiDB SQL 客户端，插入数据。随后将从示例程序读取出数据。
+本页将指导您以最快的方式开始使用 TiDB 。您将使用 [TiDB Cloud](https://en.pingcap.com/tidb-cloud) ，创建并启动一个免费的 TiDB 集群，使用 TiDB SQL 客户端，插入数据。随后将从示例程序读取出数据。
```
Comment (Oreoxmt) [inline develop/build-cluster-in-cloud.md:9]:
```suggestion
本页将指导您以最快的方式开始使用 TiDB 。您将使用 [TiDB Cloud](https://en.pingcap.com/tidb-cloud)，创建并启动一个免费的 TiDB 集群，使用 TiDB SQL 客户端，插入数据。随后将从示例程序读取出数据。
```

Comment (Icemap) [inline develop/build-cluster-in-cloud.md:9]:
```suggestion
本页将指导您以最快的方式开始使用 TiDB 。您将使用 [TiDB Cloud](https://en.pingcap.com/tidb-cloud) 创建并启动一个免费的 TiDB 集群，使用 TiDB SQL 客户端，插入数据。随后将从示例程序读取出数据。
```

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**